### PR TITLE
Add parrotschemas to reqs

### PR DESCRIPTION
**Pull request by mrknmc**

### CODE REVIEW

**Review by vetyy — APPROVED** [review]
@mrknmc you will also have to fix import in tests

File: tests/parrottools/events/publisher_test.py [hunk 1]
```diff
@@ -33,12 +33,12 @@ def test_publish_event():
                 Data=json.dumps(
                     {
                         "header": {
-                            "event_type": "events_monitoring.v1.RoughDraftStarted",
```
Comment (mrknmc) [inline tests/parrottools/events/publisher_test.py:36]: This is used in the consumer right? Should we keep it as is? `parrotschemas` can be implicit...

Comment (vetyy) [reply tests/parrottools/events/publisher_test.py:36]: yea, ideally, so we don' have to change monitoring dashboards etc

Comment (mrknmc) [reply tests/parrottools/events/publisher_test.py]: Done :+1:

**Review by vetyy — APPROVED** [review]

**Pull request merged**